src/assets/scripts/client/App.js

```diff
@@ -145,8 +141,6 @@ export default class App {
      * @method enable
      */
     enable() {
-        console.info(`openScope Air Traffic Control Simulator, Version v${this.prop.version}`);
```

Comment:
I love these little moments. 

It is available already, in the `head` of the HTML. It is included during the build and is pulled directly from the `package.json`.